tests/resources/functional/tsql/functions/test_checksum_agg/test_checksum_agg_1.sql

```diff
@@ -0,0 +1,10 @@
+## CHECKSUM_AGG
+
+There is no direct equivalent of CHECKSUM_AGG in Databricks SQL. The following
```

Comment:
@vil1 do we allow free-form text here?.. 

@jimidle could these files be "mostly valid SQL"?

Reply:
In the prelude I have started putting markdown with intent of auto producing a mini guide as we add functions, but otherwise it is free-form. That could of course use SQL comments, which would then be removed before rendering markdown. Either way is fine. Using SQl comments would mean that IDES etc can parser it as pure SQL - however, the IDE requires a dialect, so when I select TSQL, it flags a lot of Datarbicks SQL and vice-versa